src/components/ContentPreview/ContentPreview.js

```diff
@@ -343,7 +345,7 @@ class ContentPreview extends PureComponent<Props, State> {
      * @return {void}
      */
     loadPreview = (): void => {
-        const { token, collection, ...rest }: Props = this.props;
+        const { token, collection, canDownload, ...rest }: Props = this.props;
```

Comment:
For backwards compatibility, add `showDownload` in the line above, so that its not part of `rest`.
And below, pass in showDownload: canDownload || !!showDownload

Reply:
Actually might be better to make another local function in this file for now

```js
canDownload() {
    // showDownload is a prop that preview library uses and can be passed by the user
    const { showDownload, canDownload }: Props = this.props;
    return canDownload || !showDownload;
}
```



Then in the line above, use `lodash/omit` to get rid of things that we don't want sending to the preview library that the user passes in, because we are overriding those.

```js
this.preview.show(file.id, token, {
        ...omit(rest, [
                'collection',
                'showDownload',
                'skipServerUpdate',
                'header',
                'container',
                'useHotKeys'
            ])
}
```